CSS/Comments.tmPreferences

```diff
@@ -14,13 +14,13 @@
 				<key>name</key>
 				<string>TM_COMMENT_START</string>
 				<key>value</key>
-				<string>/*</string>
+				<string>/* </string>
```

Comment:
In that case, I'd like to argue how trailing spaces are allowed for line comment markers but not for blocks.

Examples:
https://github.com/sublimehq/Packages/blob/20bb6038d0fb54cc76dc4ef22e810f80fba909e9/Python/Miscellaneous.tmPreferences#L23
https://github.com/sublimehq/Packages/blob/20bb6038d0fb54cc76dc4ef22e810f80fba909e9/Java/Comments.tmPreferences#L17

I did not test how toggling commands with whitespaces in block sections behaves however, although it works fine for line comments.